src/src/PluginStructs/P128_data_struct.h

```diff
@@ -2207,8 +2207,23 @@ const uint8_t PROGMEM ftv_gamma8[] = {
 # define ARRAYSIZE 300 // Max LED Count
 
 // Choose your color order below:
+# if defined(P128_USES_GRB)
+#  define GRB
+# elif defined(P128_USES_GRBW)
+#  define GRBW
+# elif defined(P128_USES_RGB)
+#  define RGB
+# elif defined(P128_USES_RGBW)
+#  define RGBW
+# elif defined(P128_USES_BRG)
+#  define BRG
+# elif defined(P128_USES_RBG)
+#  define RBG
+# else // if defined(P128_USES_GRB)
+#  define GRB // Default
+# endif // if defined(P128_USES_GRB)
```

Comment:
This looks rather exclusive-OR like.
Can't it be made user selectable instead of defined at compile time?

Reply:
yes, that's what my intention was, but the library is some kind of C++ template 'challenge' it seems, where adding it as a user-selectable option would duplicate the entire code, and it would also require to duplicate much of the plugin code, littered with `if` statements. (I wrote that in somewhat different words, a couple of hours ago in the OP here).
Don't think it is feasible to have this, already quite 'heavy' plugin, included in that way. We could consider it as a separate PR though, if it is really needed.
I also haven't selected a default plugin-set for this plugin either, for the same reason (size). I've been thinking about a specific `NeoPixel` build-set though, holding all NeoPixel related plugins (38, 41, 42. 70, 128).

Reply:
Maybe we should then also double check if whatever we choose is working well with ESP32 IDF4.4 as there seems to be a lot of changes which needed specific libs in the past and now can be done in 1 lib for all kinds of ESP32 nodes.

Reply:
The dev-team is most likely on top of that, as even the ESP32-C3 is already supported for 6 months. Wouldn't expect many issues there.